src/components/AnimatedCheckmark.vue

```diff
@@ -58,29 +55,15 @@ const props = defineProps({
   /**
    * The color of the checkmark.
    *
-   * Valid colors are:
-   * - Available CSS-colors (for example: white, #000, etc.)
-   * - Any custom CSS-class (a-zA-Z_-).
-   *
-   * If the color is an CSS-class, this class has to set the following properties:
-   * - fill: [color]
-   * - stroke: [color]
-   *
-   * If not set, it will be white.
+   * Defaults to white.
    */
   color: { type: String, default: "white" },
 });
 
-/**
- * Returns the css-class to set for every svg-component.
- *
- * The class is the set {@link color}, if the regex /^[a-zA-Z_-]*$/ matches.
- */
-function colorClass(): string {
-  if (props.color && props.color.match(/^[a-zA-Z_-]*$/)) {
-    return props.color;
-  }
-  return "";
+function style(): string {
+  return `stroke: rgb(var(--v-theme-${props.color}));
+          fill: rgb(var(--v-theme-${props.color}));
+          stroke: ${props.color}; fill: ${props.color};`;
```

Comment:
why no computed function?

i think it works here, because there are just used props, which force the whole component to be rebuilt or something like that? But I think, a computed would be the better and possibly "correct" way to to this?

Reply:
It works because the result is never stored anywhere, just used in the component. But I'll check the computed stuff.

Reply:
Done: See https://github.com/Crystal-Creations-GbR/animated-checkmark/compare/3b1ac3e614efae78a61c4186952c896b9f4239cc..2426ba76d790534eb44846e2b31e42632aa97511